lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
Reviewers: one part of the new XSS logic that I'm not thrilled by is this kind of HTML escaping. It isn't enough to use ``<%-`` if there's a chance that you have an HTML snippet, as that will then end up being double-escaped. Instead you have to use ``<%= HtmlUtils.escape(...) %>`` so that the parameter is only escaped if necessary. Any thoughts?